lib/data/member.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:hatspace/data/data.dart';
+
+class Member {
+  final rolesKey = 'roles';
```

Comment:
@dlvinh You should specify data type for code convention and consider changing this field to private (_rolesKey), cause i think we don't need to expose it to outside.

Reply:
Thank you @DongNguyenQ, already changed